<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintModeSelect</class>
 <widget class="QWidget" name="PrintModeSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(45, 44, 43, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>20</y>
     <width>1201</width>
     <height>136</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;font-family:Barlow;background-color: rgba(45, 44, 43, 0);
font-family:Barlow;
color: rgb(255, 255, 255);
font-size: 48px;
</string>
   </property>
   <property name="text">
    <string>Please select mode:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_btnDirect">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>152</y>
     <width>1008</width>
     <height>120</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_btnDuplicate">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>1008</width>
     <height>120</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_btnMirror">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>1008</width>
     <height>120</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
  <widget class="QQuickWidget" name="qw_btnCancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>570</y>
     <width>1008</width>
     <height>120</height>
    </rect>
   </property>
   <property name="resizeMode">
    <enum>QQuickWidget::SizeRootObjectToView</enum>
   </property>
   <property name="source">
    <url>
     <string>qrc:/qml/JLPushedButton.qml</string>
    </url>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
